src/Squirrel/UpdateManager.DownloadReleases.cs

```diff
@@ -89,6 +108,30 @@ Task checksumAllPackages(IEnumerable<ReleaseEntry> releasesDownloaded)
                 return releasesDownloaded.ForEachAsync(x => checksumPackage(x));
             }
 
+            bool isPackageOk(ReleaseEntry downloadedRelease)
```

Comment:
Unless this is the squirrel coding style, do you want to capitalize the method name?